install/k8s/install.sh

```diff
@@ -100,6 +100,16 @@ EOF
 	exit 1
 }
 
+
+# this function copies $1 to $2 if the full paths to $1 and $2 (as determined by
+# `realpath`) are different.  this allows people to specify a certificate, key, etc.
+# which was moved into place by a previous installer run.
+function copy_unless_identical_paths() {
+	if [ "$1" != "$2" ]; then
```

Comment:
if you want to avoid "same src/dest" error, try
cp $1 ${1}.tmp && mv ${1}.tmp $2